cypress/support/cypress-commands/commands.js

```diff
@@ -1568,6 +1567,12 @@ Cypress.Commands.add('methodOrEventResponseValidation', (validationType, request
             ) {
               response.result = response.result.eventResponse;
             }
+            // Parse v2 events for first party events
+            if (appId === UTILS.getEnvVariable(CONSTANTS.FIRST_PARTY_APPID) && UTILS.getEnvVariable(CONSTANTS.IS_BIDIRECTIONAL_SDK)) {
+              // "result": {"key": "new value"}
+              // parse the v2 event response from {"key": "new value"} to "new value"
+              response.result = Object.values(response.result)[0];
```

Comment:
If we're expecting the request as {"key": "new value"} we should be parsing it properly. If for some reason this payload is passed with multiple keys we risk grabbing the wrong thing.